src/dynamodb/batch/batch-delete.js

```diff
@@ -36,6 +36,8 @@ async function batchDelete(
 ) {
   ensureValidParameters(documentClient, tableName, keys);
 
+  if (!keys.length) return true;
```

Comment:
Should we not check that he list has length > 0 in `ensurevalidParameters`?

Reply:
Not sure. If a client sends the result of a list.map calls that generates an empty array, do we wanna throw an error for that or just ignore it? Since this is a lib my initial thought is to ignore it, but open to changing it if that's the general consensus. @Tanax any opinion?

Reply:
I think ignoring is more common for that type of "error", no?